deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -117,7 +117,6 @@ metadata:
             "graphing": {
               "enabled": false,
               "grafana": {
-                "baseImage": "registry.redhat.io/rhel8/grafana:9",
```

Comment:
I actually need to add this back in now that I'm not using the env var setup for Grafana any more.